install.sh

```diff
@@ -1873,7 +1310,14 @@ TM_SHARED_FILES="src/tm_mad/shared/clone \
                  src/tm_mad/shared/snap_revert.ssh \
                  src/tm_mad/shared/cpds \
                  src/tm_mad/shared/cpds.ssh \
-                 src/tm_mad/shared/resize"
+                 src/tm_mad/shared/resize \
+                 src/tm_mad/shared/prebackup_live \
+                 src/tm_mad/shared/prebackup \
+                 src/tm_mad/shared/postbackup_live \
+                 src/tm_mad/shared/postbackup \
+                 src/tm_mad/shared/restore"
+
+TM_QCOW2_FILES="${TM_SHARED_FILES}"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

TM_QCOW2_FILES appears unused. Verify use (or export if used externally).